src/db.js

```diff
@@ -0,0 +1,11 @@
+import 'dotenv/config';
+import { Sequelize } from 'sequelize';
+
+export const sequelize = new Sequelize({
+  database: 'postgres',
+  username: 'postgres',
+  host: 'localhost',
+  dialect: 'postgres',
+  port: 5432,
+  password: '123456789',
```

Comment:
Sensitive data such as database name, username, host, port, and password should not be hardcoded. It's recommended to use environment variables to store such sensitive information. You can use `process.env` to access environment variables in Node.js. For example, `database: process.env.DB_NAME`. Don't forget to add these environment variables to your .env file and add .env file to .gitignore to prevent it from being committed to the repository.